federationtypes.go

```diff
@@ -444,9 +444,9 @@ type RespSendJoin struct {
 	// but not guaranteed to be present as it's only since MSC3083.
 	Event RawJSON `json:"event,omitempty"`
 	// true if the state is incomplete
-	PartialState bool `json:"org.matrix.msc3706.partial_state"`
+	PartialState bool `json:"members_omitted"`
```

Comment:
I kinda wonder if we should rename the struct field while we're here. It'll break compatibility with any apps using it, but there doesn't seem to be any advantage to delaying it?

Reply:
Not sure what the done thing is in GMSL, but I'm persuaded.

Reply:
See d5bef8c